### ISSUE TITLE
[A11Y] Issue tracker for future improvements

### ISSUE THREAD

**Issue by zorkow** [mathjax/mathjax-v3#296]
[ ] SVG magnification as discussed in PR #294. Magnification region is too large.
[ ] Maintain style sheets for regions and highlighting.

**Comment by lucad1971** [comment]
speech text export  is only for Englishlanguage...
and so also Braille code, nos upport for other languages code..
or am i missing something?

**Comment by dpvc** [comment]
> am i missing something?

Yes.  The speech text can be for more than a dozen different languages.  The "Speech" menu in the MathJax contextual menu includes a "Language" submenu that lets you select the localization of the speech text.  Note that the mathematical terms are translated, but not words for text within the math.  So something like `\sin(x^2)` will produce "der Sinus von Klammer auf x Quadrat Klammer zu" when the German locale is selected, but `\{\,x\mid \text{$x$ is divisible by 3}\, \}` will produce "die Menge aller x mit x is divisible by 3".

The Braille code is Nemeth Braille, which is not language specific, but again, any text specified in the expression will remain in its original language.

You can use the MathJax configuration object to set the locale for your web pages.  The same is true if you are using the MathJax Component framework in your own node applications.  If you are using direct access to the MathJax modules, you would need to configure the speech-rule engine directly.

The [MathJax node demos](https://github.com/mathjax/MathJax-demos-node) also provide command-line options for setting the locale.  Use the `--help` option on any of the demos to see what the options are.

**Comment by dpvc** [comment]
Also, this is the wrong place to be putting this question, @lucad1971.  This tracker is only for problems with the source repository itself.  For general problems or questions, you should use the [MathJax issue tracker](https://github.com/mathjax/MathJax/issues) instead.
